src/form/form-label-addon.style.tsx

```diff
@@ -1,13 +1,17 @@
 import styled from "styled-components";
+import { Color } from "../color";
 
 export const TriggerArea = styled.button`
     padding: 0.25rem 0.5rem 0.25rem 0.25rem;
     border: none;
     background: none;
     cursor: pointer;
 
-    svg {
+    & > svg {
```

Comment:
added default styling for form label icon to make it easier to switch out the rendered icon